source/server/active_tcp_listener.cc

```diff
@@ -325,7 +325,8 @@ void ActiveTcpListener::newConnection(Network::ConnectionSocketPtr&& socket,
 
   // If the connection is already closed, we can just let this connection immediately die.
   if (active_connection->connection_->state() != Network::Connection::State::Closed) {
-    ENVOY_CONN_LOG(debug, "new connection", *active_connection->connection_);
+    ENVOY_CONN_LOG(debug, "new connection from {}", *active_connection->connection_,
+                   active_connection->connection_->addressProvider().remoteAddress()->asString());
```

Comment:
Do we need to guard against a nullptr return from `remoteAddress`? It seems generally possible, but maybe not on any code path that end up here.

Reply:
I think we are good for this path. Shall I add an assert at least?

Reply:
Yeah, let's do that.